includes/API/WriteStoreSignedTxHandler.php

```diff
@@ -20,6 +20,43 @@
 require_once __DIR__ . "/../ApiUtil.php";
 require_once __DIR__ . "/../Util.php";
 
+function injectSignatureToPage( string $titleString, string $walletString, User $user ) {
```

Comment:
This was originally a separate function, and then moved in as a method by @JeroenDeDauw. I was wondering what is your reasoning to put it as a standalone function again, @pwirth?

Reply:
i moved it into the class as a private method due to issues i had with phpstan.
https://github.com/inblockio/DataAccounting/pull/213/commits/e254038eeac9717bb01436525b24f08639f7d5b2
But later I found out, that the "redeclaration" isseu was caused by antoher thing, so i moved it back as it was before.